### PR TITLE
Fixing a bad hint path.

### PR DESCRIPTION
**Pull request by tannergooding**
This fixes a bad hint path in BasicCompilerEmitTest.vbproj

### CODE REVIEW

**Comment by tannergooding** [comment]
FYI. @CyrusNajmabadi, @pharring 
Also, @agocke, @jaredpar, @amcasey, @ManishJayaswal 

Tricky bug that only occurred if you are doing a release build after building debug and when release has a different version than debug.

If for example you build debug with /p:OfficialBuild=false, and then build release with /p:OfficialBuild=true, then the build fails because it has ambiguous conflicts and mismatched versions (it grabs v42.42.42.42 from debug due to the hint path and should be grabbing v1.1.0.0 from release).

File: src/Compilers/VisualBasic/Test/Emit/BasicCompilerEmitTest.vbproj [hunk 1]
```diff
@@ -88,7 +88,7 @@
     </Reference>
     <Reference Include="Roslyn.Test.Utilities, Version=42.42.42.42, Culture=neutral, PublicKeyToken=fc793a00266884fb, processorArchitecture=MSIL">
```
Comment (KevinH-MS) [inline src/Compilers/VisualBasic/Test/Emit/BasicCompilerEmitTest.vbproj]: Should we delete the version info here?

**Comment by KevinH-MS** [comment]
LGTM (but we should consider removing the version info)

**Comment by pharring** [comment]
:+1:

**Comment by tannergooding** [comment]
@KevinH-MS, Meant to do that. Have corrected it to use $(AssemblyVersion).

**Comment by tannergooding** [comment]
Also see #4256

**Comment by tannergooding** [comment]
Corrected again to just remove the versioning specifics entirely.

**Comment by pharring** [comment]
:+1: again

**Pull request merged**